mlir/test/Gradient/HybridGradientTest.mlir

```diff
@@ -12,17 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// RUN: quantum-opt %s --lower-gradients=only=ps  | FileCheck %s
+// RUN: quantum-opt %s --lower-gradients=only=ps -cse | FileCheck %s
```

Comment:
Essentially it makes testing a bit cleaner because things like `llvm.mlir.addressof @enzyme_const` ops are emitted multiple times, but CSE cleans them up. Canonicalization unfortunately doesn't fold away multiple `addressof` ops.

Reply:
Got it 👍